File: leetcode-submissions/2678_Number_of_Senior_Citizens.ts
function countSeniors(details: string[]): number {
let ans:number=0;
for(let i=0;i<details.length;i++){
    let str=details[i];
    let s=str.slice(11,13);

   let digit:number=parseInt(s);
    if(digit>60){
        ans+=1;
    }

}
return ans;
};

File: leetcode-submissions/2678_Number_of_Senior_Citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int ans=0;
        for(int i=0;i<details.size();i++){
            string s=details[i];
        int digit=stoi(s.substr(11,2));
        if(digit>60){
            ans+=1;
        }

        }
        return ans;
    }
};